src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -429,16 +429,19 @@ function MoneyRequestConfirmationList({
                 text = translate('common.next');
             }
         } else if (isTypeTrackExpense) {
-            text = translate('iou.trackExpense');
+            text = translate('iou.createExpense');
```

Comment:
This code is being duplicated along with the other PR right? We'll have to take care of the conflicts.